examples/net.py

```diff
@@ -12,19 +12,21 @@
     nengo.Connection(ens, result, function=lambda x: x[0] * x[1],
                      synapse=0.01)
 
-    with nengo.Network(label='subnet') as subnet:
+    with nengo.Network(label='mysubnet') as subnet:
```

Comment:
Can we remove these changes from the PR?  It's not necessary, but no sense changing other people's examples